src/org/zaproxy/zap/extension/websocket/ui/httppanel/models/StringWebSocketPanelViewModel.java

```diff
@@ -17,12 +17,17 @@
  */
 package org.zaproxy.zap.extension.websocket.ui.httppanel.models;
 
+import java.util.Arrays;
+
+import org.apache.log4j.Logger;
 import org.parosproxy.paros.Constant;
 import org.zaproxy.zap.extension.websocket.WebSocketMessage;
 import org.zaproxy.zap.extension.websocket.utility.InvalidUtf8Exception;
 
 public class StringWebSocketPanelViewModel extends AbstractWebSocketStringPanelViewModel {
 	
+	private static final Logger logger = Logger.getLogger(StringWebSocketPanelViewModel.class);

```

Comment:
Since this is a new addition to this file shall we make it full caps as a constant?